socialBot.py

```diff
@@ -0,0 +1,76 @@
+import os
+from dotenv import load_dotenv, find_dotenv
+import openai
+from openai import OpenAI
+
+
+
+load_dotenv(find_dotenv())
+openai.api_key = os.getenv("API_KEY")
+
+# aimemory = "Question: In a coffee shop, is it good to yell loudly"
+# userMemory = "Yes it is appropriate if you need something very urgently"
+memory = "AI: In a coffee shop, is it appropriate to yell loudly? User: Yes it is appropriate if you need something very urgently"
+numAsks = 0
+
+# def generateResponse(memory, prompt):
+#     if numAsks < 10:
+#         prompt = prompt + " Here are some past responses in 10 words: " + memory + " End your response with a 10 word summary of what happened in your response so that you can use it in the future."
+#         response = llm.chat.completions.create(
+#             model="gpt-3.5-turbo",
+#             messages=[
+#                 {
+#                     {"role": "system", "content": "You are someone who can give social situations to socially awkward and autistic people, and then help critique their responses. All previous prompts and responses will be in the prompt you are given. "}, 
+#                     {"role": "user", "content": prompt}  
+#                 }
+#             ]
+#         )
+#         return response.choices[0].message.content
+#     else:
+#         print("Out of responses! Either generate a new text thread or delete this one to start fresh.")
+
+
+def ask_openai():
+    global numAsks
```

Comment:
You should look into classes. I don't want to give you too much code, but here's an example.

```
class Critiquer:
    def __init__(self):
         self.numAsks = 0
         self.memory = ""
    def ask_openai(self): # self is an implicit argument thats automatically passed when called from an instance of Critiquer. 
          [your code]
          self.numAsks += 1
          self.memory += "foobarbaz"

critiquer_instance = Critiquer()
while critiquer_instance.numAsks < 10:
    critiquer_instance.ask_openai()
```